File: solution/java/week06/game/src/main/java/order/Order.java
package order;

public class Order {
    public static final int UNPROCESSED = 0;
    public static final int PROCESSED = 1;
    private int status;
    private int items;
    private double total;

    public Order(int status, int items, double total) {
        this.status = status;
        this.items = items;
        this.total = total;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

    public int getItems() {
        return items;
    }

    public double getTotal() {
        return total;
    }

    public void setTotal(double total) {
        this.total = total;
    }
}

File: solution/ts/week06-legacy-code/exercise3/src/order.ts
export class Order {
    public static readonly UNPROCESSED = 0;
    public static readonly PROCESSED = 1;

    private status: number;
    private items: number;
    private total: number;

    constructor(status: number, items: number, total: number) {
        this.status = status;
        this.items = items;
        this.total = total;
    }

    getStatus(): number {
        return this.status;
    }

    setStatus(status: number): void {
        this.status = status;
    }

    getItems(): number {
        return this.items;
    }

    getTotal(): number {
        return this.total;
    }

    setTotal(total: number): void {
        this.total = total;
    }
}
